### PR TITLE
Explain to remove the deprecated rgbasm flags

### PR DESCRIPTION
**Pull request by Rangi42**

### CODE REVIEW

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Should we word the messages as "please avoid using [...]"?

**Comment by Rangi42** [comment]
I read it as "do not use the X flag (to fix this warning)". Also none of the other errors/warnings implore the user. :P Do you still prefer "avoid using the X flag"?

**Comment by aaaaaa123456789** [comment]
I'd avoid the command, and its associated wording problems, completely.

"Automatic `nop` after `halt` (option `-H`) is deprecated"

**Comment by Rangi42** [comment]
All the other deprecation warnings are of the form "X is deprecated; <imperative for how to fix it>".

**Comment by aaaaaa123456789** [comment]
This isn't quite the same; "don't use the deprecated feature" is redundant. "Automatic `nop` insertion after `halt` is deprecated; add them manually if needed instead of using option `-H`" would be a statement like what you say, but it seems too wordy.

**Comment by ISSOtm** [comment]
I think ax6's formulation works best here.

**Review by ISSOtm — APPROVED** [review]
Thank you!
